style.css

```diff
@@ -100,4 +104,194 @@ body {
   .social-icons .social-icon:not(:last-child) {
     margin-right: 20px;
   }
+
+  /* *---------------- Work Section -------------------  */
+  .work-section {
+    background-color: #e5e5e5;
+    padding: 114px 24px 24px 24px;
+    margin: 0;
+  }

```

Comment:
- [ ] The main container of the projects has a `display: grid` to practice using the grid template